lib/node_modules/@stdlib/blas/ext/base/dcusumkbn2/README.md

```diff
@@ -214,14 +214,14 @@ stdlib_strided_dcusumkbn2( 4, 0.0, x, 1, y, 1 );
 The function accepts the following arguments:
 
 -   **N**: `[in] CBLAS_INT` number of indexed elements.
--   **sum**: `[in] CBLAS_INT` initial sum.
+-   **sum**: `[in] double` initial sum.
```

Comment:
Same thing regarding these docs fixes. We'll want to make them in a separate `docs:` PR.